server/src/main/resources/reference.conf

```diff
@@ -19,4 +19,15 @@
     keyStorePassword=password
     keyStorePath="/path/to/your/cert"
   }
+  monitoring {
+    // monitoring of http communication
```

Comment:
comments above used `#` instead of `//` (also below)

Reply:
ok, will use # everywhere